01week/datatypes.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+// 1. program for date and time
+function returnDate() {
+  const now = new Date();
```

Comment:
The const is redundant in this case, can be simplified into one line. `return new Date();`